saleor/settings.py

```diff
@@ -964,8 +964,10 @@ def SENTRY_INIT(dsn: str, sentry_opts: dict):
 # Default timeout for external requests.
 COMMON_REQUESTS_TIMEOUT = (REQUESTS_CONN_EST_TIMEOUT, 18)
 
-WEBHOOK_TIMEOUT = (REQUESTS_CONN_EST_TIMEOUT, 18)
-WEBHOOK_SYNC_TIMEOUT = (REQUESTS_CONN_EST_TIMEOUT, 18)
+WEBHOOK_WAITING_FOR_RESPONSE_TIMEOUT = 18
+
+WEBHOOK_TIMEOUT = (REQUESTS_CONN_EST_TIMEOUT, WEBHOOK_WAITING_FOR_RESPONSE_TIMEOUT)
+WEBHOOK_SYNC_TIMEOUT = (REQUESTS_CONN_EST_TIMEOUT, WEBHOOK_WAITING_FOR_RESPONSE_TIMEOUT)
```

Comment:
Cannot it be one setting as both are the same?

Reply:
I don't have an opinion here. From one side, this could be a single env.  On the second side, having separate envs give you more flexibility. Like you would like to have a longer timeout on async webhook, as it takes some time to process the request by your app for asynchronous webhook, but you want to keep the default timeout for sync webhook.

Reply:
We agreed to keep this as it is for now. Marking as resolved